src/winforms/toga_winforms/widgets/button.py

```diff
@@ -9,10 +9,16 @@
 class Button(Widget):
     def create(self):
         self.native = WinForms.Button()
-        self.native.Click += self.winforms_click
+        self.native.Click += self.winforms_on_press
```

Comment:
The naming of `winforms_click` was deliberate. We've essentially got two signal types - the ones generated by Toga, and the ones generated by the native platform. `winforms_click` makes it clear that it's the Winforms `Click` handler; rather than needing to reverse engineer that the Toga name `on_press` is invoked by the Winforms `Click` event.

I agree the current code has a couple of inconsistencies on this pattern (e.g., should the handler be called `winforms_CheckChanged` or `winforms_check_changed`) - but broadly, I feel the Winforms handlers should reflect the Winforms event names.

Reply:
As a person reading the winforms backend code, the name `winforms_on_press` makes it clear to me that the native `Click` event is handled by the Toga `on_press` handler. Currently, I have to look up the `winforms_click` function to see which Toga event handler is called to handle native event.
However, I see that when debugging, it is more useful to see the distinction between Toga signals and native signals. So, I would like to finish up the housekeeping and rename the functions to get rid of inconsistencies.
I prefer the more pythonic underscore version: `winforms_check_changed`, and not `winforms_CheckChanged`, do you agree?

Reply:
I've renamed handlers in #44923c2